upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.12.yaml.template

```diff
@@ -240,9 +240,12 @@ data:
   # essential.
   typha_service_name: "none"
 
-  # TODO: Do we want to configure this?
   # Configure the MTU to use
-  veth_mtu: "1440"
+  {{- if .Networking.Calico.MTU }}
```

Comment:
I did not test this last commit. But if MTU is not defined in cluster spec this if is not executed, right? Because MTU is *int32 it is then nil

Reply:
I think so but we should probably confirm via a test. I can probably test this in a bit.